webui/src/pages/user/add-namespace-member-dialog.tsx

```diff
@@ -124,11 +124,10 @@ export const AddMemberDialog: FunctionComponent<AddMemberDialoProps> = props =>
                     label='Open VSX User'
                     fullWidth
                     onChange={handleUserSearch}
-                    onKeyPress={(e: React.KeyboardEvent) => {
-                        if (e.charCode === 13) {
-                            if (foundUsers.length === 1) {
-                                addUser(foundUsers[0]);
-                            }
+                    onKeyPress={(e) => {
+                        if (e.key === "Enter" && foundUsers.length === 1) {
+                            e.preventDefault();
```

Comment:
We do this to not cause any other unexpected behavior when choosing the user.

Reply:
`e.preventDefault` should be used throughout the webui for events?

Reply:
@amvanbaren it should be used for all events that we want to override and want to prevent any unexpected side effects - one example would be that natively in the browser, when you drag n' drop a file to a page, it opens, but it can be overridden by using `e.preventDefault()` and made to do actual drag n' drop uploads or similar. 

When in doubt, it's good to use it like here where browsers do some things when a user hits <kbd>Enter</kbd> on a textfield, so it "just makes sure nothing unexpected happens".

Reply:
I've opened https://github.com/eclipse/openvsx/issues/609 to further apply `e.preventDefault()` in the webui.